app/renderer/ui/components/main/sections/dataRequests/tabs/editor/index.tsx

```diff
@@ -33,7 +34,7 @@ class TabEditor extends TabComponent<any & Props> {
   // tslint:disable-next-line:prefer-function-over-method completed-docs
   public render() {
     return (
-      <div className={styles.layout}>
+      <div className={layoutStyles.layout}>
```

Comment:
Mmm, I find it a bit weird that the styles for the data requests tabs are obtained from the smart contracts section. If the layout styles or if the tab styles are common to all tabs, I would perhaps move them to another folder (`tabs-common` or something like that).

Reply:
I think that there may be already an issue related to [this](https://github.com/witnet/sheikah/issues/488), so I'm ok with merging this as is for time purposes and coming back to this issue afterwards.

Reply:
Ups sorry...that was a typo should be from the dtaa requests folder...